### PR TITLE
doc: added a new speech link and dropped `node-status` api doc.

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

File: doc/admin-api-cn.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 ===
 
 * [Route](#route)
```
Comment (moonming) [inline doc/admin-api-cn.md]: why leave this empty file?

File: README_CN.md [hunk 2]
```diff
@@ -177,9 +177,10 @@ Dashboard 默认允许任何 IP 访问。你可以自行修改 `conf/config.yaml
 
 ## 视频和幻灯片
 
-- [APISIX 的选型、测试和持续集成](https://www.upyun.com/opentalk/432.html)
-
-- [APISIX 高性能实践](https://www.upyun.com/opentalk/429.html)
+- 2019.10.30 [Apache APISIX 微服务架构极致性能架构解析](https://www.upyun.com/opentalk/440.html) .
+- 2019.8.31 [APISIX 技术选型、测试和持续集成](https://www.upyun.com/opentalk/433.html) .
+- 2019.8.31 [APISIX 高性能实战2](https://www.upyun.com/opentalk/437.html) .
+- 2019.7.6 [APISIX 高性能实战(Chinese)](https://www.upyun.com/opentalk/429.html) .
```
Comment (moonming) [inline README_CN.md]: do you have the text version?

Comment (membphis) [reply README_CN.md]: I think I can find some of them. If you have a list, please paste it.

Comment (membphis) [reply README_CN.md]: These links already contain text content ^_^

**Pull request merged**